### PR TITLE
Menu

### PR DESCRIPTION
**Pull request by ramshorst**

### CODE REVIEW

**Comment by phndiaye** [comment]
LGTM 👍

File: app/templates/components/app-header.hbs [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-          <img src='{{logo}}' />
-      {{else if  isTopbarStyle}}
-        <img src='{{randomFullLogo}}' />
-      {{else}}
-        <img src='/assets/images/upfluence-fire-white.png' />
-      {{/if}}
-    {{/link-to}}
-  </div>
+    {{#if hasLogo}}
+        <img src='{{logo}}' />
```
Comment (tgallice) [inline app/templates/components/app-header.hbs]: ident :p

File: app/templates/components/app-header.hbs [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
+    {{else if  isTopbarStyle}}
+      <img src='{{randomFullLogo}}' />
```
Comment (tgallice) [inline app/templates/components/app-header.hbs]: always usefull ?

File: app/templates/components/app-header.hbs [hunk 3]
```diff
@@ -1,16 +1,13 @@
...
-      {{else}}
-        <img src='/assets/images/upfluence-fire-white.png' />
-      {{/if}}
-    {{/link-to}}
-  </div>
+<div class='logo-container'>
+  {{#link-to 'index'}}
+    {{#if hasLogo}}
+      <img src='{{logo}}' />
+    {{else if isTopbarStyle}}
```
Comment (tgallice) [inline app/templates/components/app-header.hbs]: This exist again ?

**Pull request merged**